Documentation/ApiOverview/Environment/ApplicationContext.rst

```diff
@@ -0,0 +1,383 @@
+:orphan:
```

Comment:
Why is the page an orphan? It should be added to the menu.

Reply:
Copy&Paste from somewhere else. Removed it and linked from menu.